server/datastore/mysql/schema.sql

```diff
@@ -376,7 +376,7 @@ CREATE TABLE `host_software_installed_paths` (
   `id` bigint(20) unsigned NOT NULL AUTO_INCREMENT,
   `host_id` int(10) unsigned NOT NULL,
   `software_id` bigint(20) unsigned NOT NULL,
-  `installed_path` text COLLATE utf8mb4_unicode_ci NOT NULL,
+  `installed_path` text COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
```

Comment:
2. The second solution is to set a default for the `installed_path` column to an empty string, hoping that fixes any erroring out or hiding of rows that have an `installed_path` not set.

Reply:
What is this fixing? I don't think we need to change anything on the DB side for this bug.